.github/workflows/MacOs.yaml

```diff
@@ -0,0 +1,189 @@
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+# Continuous integration tests on macOS
+# See `Tests.yaml` for the standard suite of continuous integration tests.
+name: macOS
+
+on:
+  # This is broken, so is only run manually (see issue 2397). Refer to
+  # Tests.yaml on what to put here in order to run the tests once the
+  # build is fixed.
+  workflow_dispatch:
+    branches-ignore:
+      - gh-pages
+
+jobs:
+  # Build all test executables and run unit tests on macOS
+  unit_tests_macos:
+    name: Unit tests on macOS
+    # Don't run on `develop` since we don't require this check to pass for PR
+    # be merged. See `CONTRIBUTING.md` for details.
+    if: github.ref != 'refs/heads/develop'
+    runs-on: macos-latest
+    env:
+      # We install `openblas` instead of using the system version because we
+      # encountered this issue:
+      # https://github.com/sxs-collaboration/spectre/issues/2131
+      SPECTRE_BREW_DEPS: >-
+        cmake ccache autoconf boost openblas catch2 gsl jemalloc hdf5 pybind11
+        yaml-cpp
+      SPECTRE_PY_DEPS: numpy scipy h5py
+      SPECTRE_SPACK_DEPS: >-
+        blaze@3.5 brigand@master libxsmm libsharp~mpi~openmp
+      CHARM_VERSION: "6.8.0"
```

Comment:
Will your charm 6.10.1 PR update this and other workflows?

Reply:
yes